simtools/data_model/meta_data_model.py

```diff
@@ -0,0 +1,307 @@
+
+* type: data type (str)
+* required: boolean if this field is required as input(!)
+* default: default value
+
+These definitions are used for:
+
+* data products submitted to SimPipe ('input')
+* data products generated by SimPipe ('output')
+
+"""
+
+
+class InvalidSchemaList(Exception):
+    """
+    Exception raised for requests of unknown schema lists
+    """
+
+
+def _category_reference():
+    """
+    Metadata field REFERENCE.
+
+    Refers to the CTA Top-Level Data Model (document CTA-SPE-OSO-000000-0001.)
+
+    """
+    return {"VERSION": {"type": "str", "required": True, "default": "1.0.0"}}
+
+
+def _category_contact():
+    """
+    Metadata field CONTACT.
+
+    Organisation and person submitting or processing these data.
+
+    """
+    return {
+        "ORGANIZATION": {"type": "str", "required": True, "default": None},
+        "NAME": {"type": "str", "required": True, "default": None},
+        "EMAIL": {"type": "email", "required": True, "default": None},
+    }
+
+
+def _category_product():
+    """
+    Metadata field PRODUCT.
+
+    Describes the data product, especially its validity and associations.
+
+    """
+    return {
+        "DESCRIPTION": {"type": "str", "required": True, "default": None},
+        "CREATION_TIME": {"type": "datetime", "required": True, "default": None},
+        "ID": {"type": "str", "required": False, "default": None},
+        "DATA": {
+            "CATEGORY": {"type": "str", "required": False, "default": None},
+            "LEVEL": {"type": "str", "required": False, "default": None},
+            "TYPE": {"type": "str", "required": False, "default": None},
+            "MODEL": {
+                "NAME": {"type": "str", "required": False, "default": None},
+                "VERSION": {"type": "str", "required": False, "default": None},
+                "URL": {"type": "str", "required": False, "default": None},
+            },
+        },
+        "FORMAT": {"type": "str", "required": False, "default": "ecsv"},
+        "VALID": {
+            "START": {"type": "datetime", "required": False, "default": None},
+            "END": {"type": "datetime", "required": False, "default": None},
+        },
+        "ASSOCIATION": {"type": "instrumentlist", "required": True, "default": []},
+    }
+
+
+def _category_process():
+    """
+    Metadata field PROCESS.
+
+    Description of the process which generated these data.
+
+    """
+    return {
+        "TYPE": {"type": "str", "required": True, "default": None},
+        "SUBTYPE": {"type": "str", "required": False, "default": None},
+        "ID": {"type": "str", "required": True, "default": None},
+    }
+
+
+def _category_context():
+    """
+    Metadata field CONTEXT.
+
+    Describes list of context documents.
+
+    """
+    return {"DOCUMENT": {"type": "documentlist", "required": False, "default": None}}
+
+
+def _category_document():
+    """
+    Metadata field DOCUMENT.
+
+    Described a
```

Comment:
Missing description

Reply:
Copy and paste error - the last line of the docstring is not needed.